src/libs/WorkspacesUtils.ts

```diff
@@ -0,0 +1,223 @@
+import Onyx from 'react-native-onyx';
```

Comment:
So I feel like this libs file is a bit redundant as it. This should either be moved to `PolicyUtils` or maybe rename this to WorkspaceSettingsUtils, however i still feel like this can be mainly in Policy utils, we can refactor this later though.

Please do change the name at least for now though